com.unity.cinemachine/Editor/Editors/CinemachineGroupComposerEditor.cs

```diff
@@ -9,10 +9,11 @@ namespace Cinemachine.Editor
 {
     [Obsolete]
```

Comment:
No point making cosmetic fixes on this file, it's marked obsolete and will be moved to a deprecated folder